### ISSUE TITLE
Could not load file or assembly 'FSharp.Core'

### ISSUE THREAD

**Issue by JonCanning** [LambdaFactory/Neptune-public#21]
My Expecto tests run from the command line, but when I use Neptune I see this message in the output window

```
Can't find custom attr constructor image: /Users/Jon/Dev/Dragonfly/tests/bin/Debug/netcoreapp2.1/Tests.dll mtoken: 0x0a000001 due to: Could not load file or assembly 'FSharp.Core, Version=4.4.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. assembly:FSharp.Core, Version=4.4.3.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a type:<unknown type> member:(null) signature:<none>

Unhandled Exception:
System.IO.FileNotFoundException: Could not load file or assembly 'Expecto, Version=8.1.1.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies.
File name: 'Expecto, Version=8.1.1.0, Culture=neutral, PublicKeyToken=null'
```

**Comment by Krzysztof-Cieslak** [comment]
Can you please check if that has been fixed with latest **Ionide** release (`3.22.4`) ?

**Comment by JonCanning** [comment]
Yes it has! Thanks! 🎉
